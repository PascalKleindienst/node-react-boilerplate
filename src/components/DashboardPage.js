// ==============================================
// DASHBOARD PAGE COMPONENT =====================
// ==============================================
import React from 'react';

const DashboardPage = () => (
    <div>
        Dashboard page content
    </div>
);

export default DashboardPage;
