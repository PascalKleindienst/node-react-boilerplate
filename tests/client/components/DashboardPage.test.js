// ==============================================
// DASHBOARD PAGE TEST ==========================
// ==============================================
import React from 'react';
import { shallow } from 'enzyme';
import DashboardPage from '../../../src/components/DashboardPage';

describe('my dashboard page component', () => {
    test('should render correctly', () => {
        const wrapper = shallow(<DashboardPage />);
        expect(wrapper).toMatchSnapshot();
    });
});
