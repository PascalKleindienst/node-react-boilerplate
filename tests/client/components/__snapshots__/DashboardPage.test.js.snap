// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`my dashboard page component should render correctly 1`] = `
<div>
  Dashboard page content
</div>
`;
